cmd/dlt/autoscaler/cmd.go

```diff
@@ -62,10 +62,6 @@ func DeleteAutoscalerRunner() rosa.CommandRunner {
 			return err
 		}
 
-		if cluster.Hypershift().Enabled() {
```

Comment:
for hcp autoscaler, we only support edit/get autoscaler. create/dlt are not supported